Print styling

KyraWayne opened the pull request:
will style printing for readability

Commit:
will print only calendar and list of classes left

I made a lot of unnecessary things hidden, kept the percentage of
progress, calendar, and a list of the unscheduled classes

Commit:
will simplify quarter drop down

easier

Commit:
working on js for print

Commit:
Merge branch 'master' into print-styling

Commit:
Merge branch 'master' into print-styling

Commit:
will style printing for readability

hides unnecessary items, leaves calendar and list of unscheduled classes

Commit:
Merge branch 'master' into print-styling

jkingsman commented on css/style.css:
```diff
@@ -76,3 +80,11 @@ th {
+
+h5{
+    display:none;
+    visibility:hidden;
```
Could we add a custom class or something to this? Making an unguarded `display: none` on a standard style seems dangerous to me.

KyraWayne replied:
added .print-titles class

jkingsman committed:
Merge branch 'master' into print-styling

jkingsman commented on js/print.js:
```diff
@@ -1,3 +1 @@
-    document.getElementsByClass
-}*/
\ No newline at end of file
+$('.print-btn').on('click',$('ul.unscheduledList li').append(', '));
```
Is this still needed with the CSS work you've done?

KyraWayne replied:
deleted the file

jkingsman reviewed (changes requested):
Looks good; a couple questions and 1 or 2 changes requested but all minor.

Commit:
adds print-titles class and deletes new js

no need for print.js and adds class .print-titles to items only visible
during printing
